File: app/src/main/java/fr/jbme/raiderioapp/data/model/character/Gear.java
package fr.jbme.raiderioapp.data.model.character;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Gear {

    @SerializedName("item_level_equipped")
    @Expose
    public Integer itemLevelEquipped;
    @SerializedName("item_level_total")
    @Expose
    public Integer itemLevelTotal;
    @SerializedName("artifact_traits")
    @Expose
    public Float artifactTraits;
    @SerializedName("corruption")
    @Expose
    public Corruption corruption;
    @SerializedName("items")
    @Expose
    public Items items;

}

File: app/src/main/java/fr/jbme/raiderioapp/data/model/character/Gear.kt
package fr.jbme.raiderioapp.data.model.character

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class Gear(
    @SerializedName("item_level_equipped")
    @Expose
    var itemLevelEquipped: Int? = null,

    @SerializedName("item_level_total")
    @Expose
    var itemLevelTotal: Int? = null,

    @SerializedName("artifact_traits")
    @Expose
    var artifactTraits: Float? = null,

    @SerializedName("corruption")
    @Expose
    var corruption: Corruption? = null,

    @SerializedName("items")
    @Expose
    var items: Items? = null
)